account_cutoff_start_end_dates/__init__.py

```diff
@@ -1 +1,45 @@
 from . import models
+
+from openupgradelib import openupgrade
+from psycopg2 import sql
+
+
+def module_migration(cr):
+    if openupgrade.is_module_installed(cr, "account_cutoff_start_end_dates"):
+        return
+
+    account_cutoff_line_table = "account_cutoff_line"
+    if openupgrade.column_exists(cr, account_cutoff_line_table, "prepaid_days"):
+        openupgrade.update_module_names(
```

Comment:
The existence of column `prepaid_days` does not necessarily mean that `account_cutoff_prepaid` was installed because it might have been created from any module; if you want to rename the module I think it is better to check directly if it was installed using `is_module_installed`.

Moreover, from my understanding of https://github.com/OCA/account-closing/pull/153, this new module `account_cutoff_start_end_dates` also substitutes the module `account_cutoff_accrual_dates` so `update_module_names` should be executed on that too, otherwise the following warning might still show in the log after `account_cutoff_start_end_dates` installation:
> odoo.modules.graph: module account_cutoff_accrual_dates: not installable, skipped

Reply:
ok thanks, I'll modify that too